CHANGELOG.md

```diff
@@ -8,16 +22,19 @@
 
 ## Tink PFM 0.9.7
 - Added multiple styling options: 
-- Configure the check box radius with `checkBoxIconBackgroundCornerRadiusFactor` in `ColorProvider`.
-- Configure inverted button with `secondaryButtonTitle` in `ColorProvider`.
-- Configure pill button with `tertiaryButtonTitle` and `tertiaryButtonBackground` in `ColorProvider`.
-- Configure monthly picker chevron button with `chevronButton` and `chevronButtonDisabled` in `ColorProvider`.
-- Configure to show currency symbol on overview with `shouldShowCurrencySymbolInStatisticsTotal` in `FinanceOverviewViewController`
-- Configure `UISegmentedControl` appearance with appearance proxy.
-- Updated date formats on transactions.
+    - Configure the check box radius with `checkBoxIconBackgroundCornerRadiusFactor` in `ColorProvider`.
+    - Configure inverted button with `secondaryButtonTitle` in `ColorProvider`.
+    - Configure pill button with `tertiaryButtonTitle` and `tertiaryButtonBackground` in `ColorProvider`.
+    - Configure monthly picker chevron button with `chevronButton` and `chevronButtonDisabled` in `ColorProvider`.
+    - Configure to show currency symbol on overview with `shouldShowCurrencySymbolInStatisticsTotal` in `FinanceOverviewViewController`
+    - Configure `UISegmentedControl` appearance with appearance proxy.
+- Updated date formats on transactions - Always Italian locale for date formatting on Enel-X app.
```

Comment:
Should this be here?

Reply:
I think you are checking some outdated commit